### PR TITLE
Errors and  Typo removal

### PR DESCRIPTION
**Pull request by ChaiBapchya**
@dcwalk @b5 
Given pull request consists of following commits
## 1. web_monitoring/db.py
+ Problem - SyntaxError: invalid syntax
+ Solution - comment empty classes and ...

## 2. requirement.txt
With reference to edgi-govdata-archiving/web-monitoring-processing#26
+ Typo - alchemy

## 3. web-monitoring/app.py
+ Name Error
```
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
/home/chai/GSOC/web-monitoring-processing/web_monitoring/app.py in <module>()
     15 MONGO_DB_URI = 'mongodb://localhost:27017/'
     16 MONGO_DB_NAME = 'page_freezer_v1'
---> 17 engine = sqlalcehmy.create_engine(SQL_DB_URI)
     18 client = pymongo.MongoClient(MONGO_URI)
     19

NameError: name 'sqlalcehmy' is not defined
```
+ Solution - correct the typo _sqlalcehmy_ to *sqlalchemy*

## 4. Readme.md
+ Add refer developer docs.md for contributing to this repository

## 5. Developer docs

+ Typo - page displays **viral** statistics (corrected to **vital**)

+ Development install - Added pre-requisite of Download+Install **_PostgreSQL_**

### CODE REVIEW

File: DEVELOPER_DOCS.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ When a user shows up:
 * Server determines the highest priority diff that needs human inspection and
   redirects user to ``/diff/<DIFF_HASH>``. This is a permanent link that can be
   shared or revisited later.
-* The page at ``/diff/<DIFF_HASH>`` displays viral statistics about the diff,
+* The page at ``/diff/<DIFF_HASH>`` displays vital statistics about the diff,
```
Comment (danielballan) [inline DEVELOPER_DOCS.md]: Haha. Good catch!

File: web_monitoring/db.py [hunk 2]
```diff
@@ -434,18 +434,18 @@ def diff_version(version_uuid, versions, diffs, source_type,
                  source_metadata)
 
 
-class WebVersioningException(Exception):
+#class WebVersioningException(Exception):
```
Comment (danielballan) [inline web_monitoring/db.py]: Is this change intentional? At least some of this are used in the module, so I believe commenting them out will break things.

File: DEVELOPER_DOCS.md (hunk 1 above)
Comment (ChaiBapchya) [inline DEVELOPER_DOCS.md]: Thanks..:D

File: web_monitoring/db.py (hunk 2 above)
Comment (ChaiBapchya) [inline web_monitoring/db.py]: The problem is
Just commenting the '...' would leave the class empty
I commented the Empty class that's it.
It doesn't give error since hasn't been referred.
Anyway if needed, I'll uncomment that part. @danielballan

**Comment by danielballan** [comment]
@ChaiBapchya Cool, thanks. The purpose of those "empty classes" is to serve as custom exceptions. Since they inherit from `Exception` and add no new methods or attributes, I assume you are wondering, "Why not just use `Exception` itself?" This pattern allows code to filter specific errors and react accordingly.
